pkg/network/protocols/kafka/statkeeper.go

```diff
@@ -71,9 +71,9 @@ func (statKeeper *StatKeeper) GetAndResetAllStats() map[Key]*RequestStat {
 
 func (statKeeper *StatKeeper) extractTopicName(tx *KafkaTransaction) string {
 	// Limit tx.Topic_name_size to not exceed the actual length of tx.Topic_name
-	if tx.Topic_name_size > uint16(len(tx.Topic_name)) {
+	if uint16(tx.Topic_name_size) > uint16(len(tx.Topic_name)) {
```

Comment:
Shouldn't you cast `len(tx.Topic_name)` to `uint8` instaed?

Reply:
to ensure we're not making a mistake by increasing topic_name ubffer beyond 255, I use uint16 cast hereto ensure we're not having "buffer overflow here"